packages/core/test/integration/sequelize/transaction.test.ts

```diff
@@ -122,12 +146,14 @@ describe(getTestDialectTeaser('Sequelize#transaction'), () => {
             {
               transaction: transaction1,
               nestMode: TransactionNestMode.separate,
-              type: TransactionType.EXCLUSIVE,
+              type: sequelize.dialect.supports.startTransaction.transactionType
+                ? TransactionType.EXCLUSIVE
+                : undefined,
               isolationLevel: IsolationLevel.READ_UNCOMMITTED,
               constraintChecking: sequelize.dialect.supports.constraints.deferrable
                 ? ConstraintChecking.DEFERRED
                 : undefined,
-              readOnly: true,
+              readOnly: sequelize.dialect.supports.startTransaction.readOnly,
```

Comment:
this should be reverted back to `true`, as this option must be supported by all dialects

Reply:
Update now.